XamarinCommunityToolkit/Views/CameraView/MediaCapturedEventArgs.shared.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class MediaCapturedEventArgs : EventArgs
```

Comment:
Since this is an EventArgs all properties should be read-only. What do you think? If yes I can change it on PR if needed

Reply:
Changed this to have a internal constructor we can use, the rest is read-only. Good call!